docs/_includes/sitemap.html

```diff
@@ -1,5 +1,12 @@
 <ul>
     {% for item in include.map %}
+    {% if item.subfiles.size == 1 %}
+        <div>
+            <button class = "single">
+            <a class = "single-items" href="{{site.baseurl}}{{ files.url }}" >{{ item.title }}</a>
```

Comment:
does `files.url` still work without the files variable that was in the for loop?
also shift identation